cam_status.py

```diff
@@ -0,0 +1,82 @@
+import requests
+import argparse
+import sys
+import json
+from datetime import datetime, timedelta
+import time
+
+class test:
+    #Set up workspace for API calls to Rhombus Systems
+    def __init__(self,args):
+        #Initialize argument parser and parse command line arguments
+        self.argParser = self.__initArgParser()
+        self.args = self.argParser.parse_args(args)
+        #Create a session to set default call Header information. 
+        #This verifies to the Rhombus servers that we have authorization to access the API
+        self.session = requests.session()
+        self.session.headers = {"x-auth-scheme": "api-token","x-auth-apikey": self.args.apiKey}
+        #Set a default base URL for api calls
+        self.url = "https://api2.rhombussystems.com/api/"
+    def getStatus(self):
+        #Get a list of all cameras in the org
+        sysStatus = self.session.post(self.url+"camera/getMinimalList")
+        #Check status code to ensure that the expected response was received
+        if sysStatus.status_code != 200:
+            print("Error in /api/camera/getMinimalList")
+            return 0
+        else:
+            #print formating details for this scripts output
+            print("CAMERA NAME (UUID): STATUS - STATUS DETAILS")
+            #convert the response into a easily traversable list
+            parseSysStatus = json.loads(sysStatus.text)
+            parseSysStatus = parseSysStatus["cameras"]
+
+            for camera in parseSysStatus:
+                uuid = camera["uuid"]
+                if(self.args.uuid): #If a camera uuid was specified
+                    if(self.args.uuid != uuid): #If the current uuid is not equivalent to the expected uuid
+                        continue #skip to next camera without storing other data
+                status = camera["healthStatus"]
+                detail = camera["healthStatusDetails"]
```

Comment:
This can be just None, which prints awkwardly.  I would consider omitting this completely from the print line for a cleaner output.

Reply:
Update: I changed the output to omit None, but keep other details

`Michigan - Camera 1(VJ9igmMwRsCDiimKW2gUIw): GREEN`
`Rhombus HQ - Camera 1(IdQWkWO4S56_BgTJCYzQdA): RED - DISCONNECTED`
`        Offline since: 07/21/21, 11:42`
`        Down for: 5 days, 0:10:39.419999`